src/plugins/vis_type_vega/public/expressions/vega_fn.ts

```diff
@@ -52,6 +52,14 @@ interface Arguments {
 
 export type VisParams = Required<Arguments>;
 
+export type VegaExpressionFunctionDefinition = ExpressionFunctionDefinition<
```

Comment:
if sticking with the `line` definitions, maybe change this to `VegaLineExpressionFunctionDefinition`?